src/lib/review/keys.ts

```diff
@@ -0,0 +1,19 @@
+import type { MonthlySummaryFilters } from './types';
+
+export const reviewKeys = {
+  all: ['review'] as const,
+  monthlySummary: (filters: MonthlySummaryFilters = {}) =>
+    [
+      ...reviewKeys.all,
+      'monthly-summary',
+      filters.dateFrom,
+      filters.dateTo,
+      filters.districts,
+    ] as const,
+  districts: () => [...reviewKeys.all, 'districts'] as const,
```

Comment:
Suggestion: Remove the districts key because the monthly summary already returns the list of available districts